src/install_pypi/plan/cache.rs

```diff
@@ -0,0 +1,233 @@
+//! Cache resolution for distribution packages.
```

Comment:
This is both a rename and now contains just a single version that handles the cache logic.